### DISCUSSION TITLE
How to set Virtual Worker in using Pysyft

### DISCUSSION THREAD

**Discussion by aamar-shahzad** [OpenMined/courses#357] [General]
I want to train the model on the virtual workers by creating a torch hook. but i can not able to create a virtual worker because the syft.virtualWorker function is no more available in pytorch.

**Comment by hgascon** [comment]
Did you find the necessary documentation to solve this issue in the last version of syft?

**Reply by yemikifouly** [reply]
Hello @hgascon! If you have questions about this, please post in the #support channel of our [Slack workspace](https://openmined.slack.com). 🙂

**Comment by aksheetha** [comment]
Hello I want to join the slack channel can you please send me an invite?
